www/css/transition.css

```diff
@@ -0,0 +1,69 @@
+/* COMMENT OUT ALL THE SCALE PARTS FOR A SMOOTH HORIZONTAL ANIMATION */
+.fadeOut_R{
```

Comment:
Since we're not using a fade out, but a slide transition, can you rename these functions .slideOut_R and .slideInR, etc.?

Also, add spaces before all curly brackets here and below, e.g.: `.slideOut_R {`

Reply:
Ok .. I will do it